homeworks/andriy.shtanko_Andrey-Shtanko/3-functions/functions.js

```diff
@@ -0,0 +1,37 @@
+const OIL = 926;
+const WATER = 1000;
+const MERCURY = 1355;
+
+const liquidValue = 1;
+
+const oilCount = 3;
+const waterCount = 5;
+const mercuryCount = 2;
+
+const getWeight = (weight, liquid, count) => {
+    const resultWeightOfProduct = weight * liquid * count;
+    return resultWeightOfProduct;
```

Comment:
Дуже довга назва + створення константи тут не потібно
```suggestion
    return weight * liquid * count;
```

Reply:
Все пофікшено в останньому комміті